src/display/annotation_layer.js

```diff
@@ -3022,7 +3022,9 @@ class AnnotationLayer {
 
     this.#setAnnotationCanvasMap();
 
-    await this.l10n.translate(layer);
+    if (typeof PDFJSDev === "undefined" || !PDFJSDev.test("MOZCENTRAL")) {
```

Comment:
[Given this code](https://github.com/mozilla/pdf.js/blob/2a3090224fa60110ad80097002beb42d0f5547cb/src/display/annotation_layer.js#L2918-L2923) I'd kind of expect the following condition here (and similar elsewhere in this patch).
```suggestion
    if (
      typeof PDFJSDev !== "undefined" &&
      PDFJSDev.test("GENERIC && !TESTING") &&
      this.l10n instanceof NullL10n
    ) {
```